src/olmo_core/nn/rope.py

```diff
@@ -199,20 +369,27 @@ def _get_rotary_embedding(
             return pos_sin[:seq_len, :], pos_cos[:seq_len, :]
 
         with torch.autocast(device.type, enabled=False):
-            inv_freq = 1.0 / (
-                self.theta
-                ** (torch.arange(0, self.dim, 2, device=device, dtype=torch.float) / self.dim)
-            )
-            if self.scaling is not None:
-                inv_freq = self.scaling.scale_inv_freq(inv_freq)
+            if self.scaling is None:
+                inv_freq = 1.0 / (
+                    self.theta
+                    ** (torch.arange(0, self.dim, 2, device=device, dtype=torch.float) / self.dim)
+                )
```

Comment:
I think the helper function you added does the same thing, right?
```suggestion
                inv_freq = compute_inv_freqs(self.theta, self.dim, device)
```

Reply:
yes missed this one, thanks!